src/app/api/chapter/[chapterId]/route.ts

```diff
@@ -31,5 +42,40 @@ export const DELETE = withSessionAndRole(["ADMIN"], async ({ params }) => {
     )
   );
 
-  return NextResponse.json("ok");
+  await prisma.chapterRequest.delete({
+    where: {
+      id: chapter.chapterRequestId,
+    },
+  });
```

Comment:
```suggestion
```

This can be removed because in theory the onDelete cascade should take care of this